src/index.html

```diff
@@ -1,13 +1,281 @@
+      <section class="main__store store" id="store">
+        <h2 class="store__title">
+          Browse Nothing products by category
+        </h2>
+
+        <div class="store__categories">
+          <div class="store__category">
+            <div class="store__pictures">
+              <div class="store__picture store__picture--parrots"></div>
+              <div class="store__picture store__picture--technics"></div>
+            </div>
+            All products
+          </div>
+
+          <div class="store__category">
+            <div class="store__pictures">
+              <div class="store__picture store__picture--earplug"></div>
+              <div class="store__picture store__picture--earpods"></div>
+            </div>
+            Audio
+          </div>
+
+          <div class="store__category">
+            <div class="store__pictures">
+              <div class="store__picture store__picture--phone"></div>
+              <div class="store__picture store__picture--charger"></div>
+            </div>
+            Accessories
+          </div>
+        </div>
+
+        <div class="store__view-all">
+          View all
+        </div>
+      </section>
+
+      <div class="main__backgroundSection"></div>
+
+      <section class="main__about about">
+        <div class="about__title">
+          About us
+        </div>
+
+        <div class="about__description">
+          <p class="about__paragraph">
+            We're a London-based tech company on a mission
+            to remove barriers between people and technology.
+          </p>
+          <p class="about__paragraph">
+            To do this, we're crafting intuitive, flawlessly connected
+            products that improve our lives without getting in the way.
+            No confusing tech-speak. No silly product names.
+            Just artistry, passion and trust. And products we're proud
+            to share with our friends and family. Simple.
+          </p>
+          <p class="about__paragraph">
+            Because beautiful tech shouldn't be complicated.
+          </p>
+        </div>
+      </section>
+    </main>
+
+    <footer class="page__footer footer">
+      <section class="footer__contact-us contact-us">
+        <h2 class="contact-us__title">
+          Contact us
+        </h2>
+
+        <div class="contact-us__wrap">
+          <form
+            action="#"
+            class="contact-us__form"
+            onsubmit="this.reset(); return false;"
+          >
+            <input name="name"
+              class="contact-us__input"
+              placeholder="Name"
+              type="text"
+            >
+            <input
+              name="email"
+              class="contact-us__input"
+              placeholder="Email"
+              type="text"
+            >
+            <textarea
+              name="textarea"
+              class="contact-us__textarea"
+              placeholder="Message"
+            >
+            </textarea>
```

Comment:
```suggestion
            <textarea
              name="textarea"
              class="contact-us__textarea"
              placeholder="Message"
            ></textarea>
```

Fix format to remove extra spaces in textarea field